share/doc/homebrew/Tips-N'-Tricks.md

```diff
@@ -27,6 +27,15 @@ This can be useful if a package can't build against the version of something you
 
 And of course, you can simply `brew link $FORMULA` again afterwards!
 
+## Activate a previously installed version of a formula
+
+```sh
+brew info $FORMULA
+brew swtich $FORMULA $VERSION
```

Comment:
Thank you!

On Wed, Nov 4, 2015 at 1:01 PM, Misty De Meo <notifications@github.com>
wrote:

> In share/doc/homebrew/Tips-N'-Tricks.md
> <https://github.com/Homebrew/homebrew/pull/45694#discussion_r43915155>:
>
> > @@ -27,6 +27,15 @@ This can be useful if a package can't build against the version of something you
> >
> >  And of course, you can simply `brew link $FORMULA` again afterwards!
> >
> > +## Activate a previously installed version of a formula
> > +
> > +```sh
> > +brew info $FORMULA
> > +brew swtich $FORMULA $VERSION
>
> switch, not swtich!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/45694/files#r43915155>.
>